### ISSUE TITLE
The State of EZ?

### ISSUE THREAD

**Issue by ambiennt** [codehz/ElementZero#167]
A lot of people in the ElementZero community, including myself are concerned for the future of this software. As many people in this community have observed, updates and word from CodeHZ himself have been virtually nonexistent since late august. ElementZero is very powerful and useful for server owners, and many people have oriented their own servers around this software. Some members close to the community have tried to get EZ walking in 1.16.100 but have yet to be successful. Even if a successful build was created by other members of the community, we suspect many of the mods will be broken. 

The main question I and many others have is: Where is HZ? Is he abandoning this project? We would really appreciate a response as to what is going to happen with this amazing software- hopefully it doesn't have to die.

**Comment by elijahr2411** [comment]
He **has** been online, he approved my pr yesterday. However, his last commit was in august. So I'm wondering this also.

**Comment by Weissnix4711** [comment]
Yes. If you look at his public history, reviewing your PR is the only thing EZ-related he's done in a while.

**Comment by elijahr2411** [comment]
It'd be a shame if the project was abandoned. Today I put 2+ hours into the website

**Comment by elijahr2411** [comment]
Although @codehz hasn't been very active lately, Several members of the community (Namely @Extollite, @Weissnix4711, and sometimes myself) are working at keeping ElementZero up to date.

**Comment by Weissnix4711** [comment]
*Trying to keep EZ up to date.

**Comment by elijahr2411** [comment]
@Weissnix4711 lol

**Comment by ambiennt** [comment]
2 weeks passed and no word from HZ. Things are looking grim ;(

**Comment by qWertu133** [comment]
EZ will die If mods will not be updated , and it slowly dies right now, pretty sad

**Comment by hteppl** [comment]
> EZ will die If mods will not be updated , and it slowly dies right now, pretty sad

bdsx

**Comment by qWertu133** [comment]
> > EZ will die If mods will not be updated , and it slowly dies right now, pretty sad
> 
> bdsx
Maybe. Im waiting for normal anti-cheat on bdsx or updating of  EZ, Im not a programmer , shitty

**Comment by ambiennt** [comment]
I think BDSX is fine, but a lot of the people have already made a foundation on EZ. It is more powerful and gives us more control, which is a big appeal. Some server owners even have custom scripts and mods that only work with EZ, like myself. This makes me really hope that @codehz comes back to the project, especially with 1.16.200 coming around the corner.

**Comment by qWertu133** [comment]
meh It seems more and more what he decided to leave this project on very long time, or leave it completely rlly

**Comment by qWertu133** [comment]
weissnix and extollite was my hope xd

**Comment by ambiennt** [comment]
I mean all we need is confirmation

**Comment by qWertu133** [comment]
maybe.... but sometimes  ignoring - it is the best confirmation , unfortunately

**Comment by zxl777** [comment]
What problems do you use EZ plugins or BDX mainly to solve?
Is it to avoid hackers cheating and destroying servers?

```
I developed a V86 app for deploying and managing Minecraft Bedrock servers.
Since I couldn't wait for the update of EZ, I used Wine64+BDS for Windows to solve the 1.6.1101 problem. 
BDS for Windows has not been modified, so the stability is very good.
```
